scripts/irods/test/test_resource_types.py

```diff
@@ -402,6 +402,11 @@ def test_local_iput_with_force_and_destination_resource__ticket_1706(self):
         os.remove(doublefile)
 
 
+    @unittest.skip('This test only applies to compound hierarchies.')
+    def test_iget_data_object_as_user_with_read_only_access_and_replica_only_in_archive__issue_6697(self):
+        pass
```

Comment:
Are you planning on implementing these or do they exist for some other reason?

Just curious.

Reply:
I put the test in `resource_suite` and that means that every test class in `test_resource_types` will run the test unless it is overridden and skipped in the appropriate test classes. So, we could either copy the test for all the test classes that we want to run this test, or we can do it this way.

I'm fine with either way.

Reply:
Oh, this is foundational code.

Derived classes can choose to override the implementation if they need to.

Reply:
Correct. Resolving this since it is common practice in the resource test suite.